app/src/shared/components/ButtonToggle/index.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+import React, { useState, useCallback } from 'react'
+import cx from 'classnames'
+
+import styles from './buttonToggle.pcss'
+
+type Props = {
+    options: Array<string>,
+    defaultSelectedOption?: string,
+    className?: string,
+    onChange?: (string) => void,
+}
+
+const ButtonToggle = ({ options, defaultSelectedOption, className, onChange }: Props) => {
```

Comment:
No way to update the selected value from outside, and changes to the `defaultSelectedOption` prop don't get reflected in the internal state.

This is basically a `select` or set of radio buttons with a slightly different appearance. Consider copying the API of a native element as much as possible, or even just using a native element and swapping out the visuals. Native will be accessible and support standard keyboard navigation by default.